Packs/KnowBe4_KMSAT/ModelingRules/KnowBe4KMSATEventCollector_1_3/KnowBe4KMSATEventCollector_1_3.xif

```diff
@@ -0,0 +1,11 @@
+[MODEL:model="Audit", dataset="knowbe4_kmsat_raw"]
```

Comment:
Hi, no need to add model="Audit" if this is the ONE DM.
[MODEL:  dataset="knowbe4_kmsat_raw"]